driver/registry_sql.go

```diff
@@ -38,7 +36,7 @@ func init() {
 }
 
 type schemaCreator interface {
-	CreateSchemas() (int, error)
+	CreateSchemas(driver string) (int, error)
```

Comment:
Maybe rename this to `dbName`, otherwise this would be considered the driver which would be expected to be `postgres` for CockroachDB.